tests/validate_schema_test.go

```
@@ -0,0 +1,118 @@
+	assert.NoError(t, err)
+	os.Stdout = oldStdout
+	exitCalled := false
+	utils.OsExit = func(code int) {
+		exitCalled = true
+		if code != 1 {
+			t.Errorf("Expected exit code 1, got %d", code)
+		}
+	}
+	utils.PrintErrorMarkdownAndExitFn = func(title string, err error, suggestion string) {}
+	os.Args = []string{"atmos", "validate", "schema"}
+	err = cmd.Execute()
```

golangci-lint (1.64.8)

[error] ineffectual assignment to err

(ineffassign)